.travis/release.sh

```diff
@@ -9,8 +9,10 @@ fi
 BASE_BUILD_IMAGE=${BASE_BUILD_IMAGE:-"jaegertracing/jaeger-operator"}
 OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
 OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d\.]+)")
+JAEGER_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)")
 TAG=${TAG:-"v${OPERATOR_VERSION}"}
 BUILD_IMAGE=${BUILD_IMAGE:-"${BASE_BUILD_IMAGE}:${OPERATOR_VERSION}"}
+CREATED_AT=$(date +%FT%TZ)
```

Comment:
This doesn't look right:

```console
$ date +%FT%TZ
2019-04-03T17:55:53Z
$ date -u +%FT%TZ
2019-04-03T15:55:55Z
```

The first should have been the UTC time, as its timezone is "Z", making it off by two hours when executed on my machine (CEST), as seen on the second command.

Perhaps a more correct version could be:

```console
$ date -u -Isecond
2019-04-03T15:56:38+00:00
```

Reply:
Can't find anything that specifies the format they want for the field, so I guess your last example could be used. I'll update.